src/utils/convertMarkdown.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { Fragment } from "react";
+
+import ReactMarkdown from "react-markdown";
+import glm from "remark-gfm";
+import emoji from "remark-emoji";
+import externalLinks from "remark-external-links";
+import sanitize from "rehype-sanitize";
+
+export const convertMarkdown = (
+  text: string | undefined,
+  options?: { images?: boolean; allowP?: boolean }
+) => {
+  if (!text) return;
+  const disallowed: Array<string> = [];
+  if (options?.images === false) disallowed.push("img");
+  return text.split("\\n").map((word, index) => {
```

Comment:
As I think I asked in a previous review, what is the purpose of this `text.split("\\n").map((word, index)` here?

Reply:
@margulies do you remember? ;)

Reply:
I removed that for now.